include/secp256k1_musig.h

```diff
@@ -89,8 +89,7 @@ typedef struct {
  *
  * Fields:
  *   present: indicates whether the signers nonce is set
- *     index: index of the signer in the MuSig; set to be consistent with the order
- *            of the pubkeys provided to `secp256k1_musig_pubkey_combine`
+ *     index: index of the signer in the MuSig
```

Comment:
Can you say "index of the signer in the MuSig key aggregation"? In the threshold case there will be a potential ambiguity, since there may be an incomplete list of signers which a user may believe requires re-indexing.

Reply:
fixed